ecosystem/sep-0006.md

```diff
@@ -987,7 +987,8 @@ Name | Type | Description
 `stellar_transaction_id` | string | (optional) transaction_id on Stellar network of the transfer that either completed the deposit or started the withdrawal.
 `external_transaction_id` | string | (optional) ID of transaction on external network that either started the deposit or completed the withdrawal.
 `message` | string | (optional) Human readable explanation of transaction status, if needed.
-`refunded` | boolean | (optional) Should be true if the transaction was refunded. Not including this field means the transaction was not refunded.
+`refunded` | boolean | (**deprecated**, optional) This field is deprecated in favor of the `refunds` object. Should be true if the transaction was refunded in full. Partial refunds can still be expressed using the `refunds` object.
```

Comment:
Marking this field as deprecated is still ambiguous to me. What are our plans with the field? For backwards compatibility we are still likely to support it indefinitely since removing support seems like it would have very little value. Do we need to deprecate it?